### PR TITLE
planner, executor: return err when INSERT/UPDATE/ANALYZE/DELETE a sequence

### PR DESCRIPTION
**Pull request by AilinKid**
### What problem does this PR solve?

Problem Summary: return err when INSERT/UPDATE/ANALYZE/DELETE a sequence

### What is changed and how it works?

What's Changed:
 judge table whether is a sequence when INSERT/UPDATE/ANALYSE/DELETE and return error.

### Check List 

Tests 

- Unit test
- Integration test

### Release note 
planner, executor: return err when INSERT/UPDATE/ANALYZE/DELETE a sequence

### CODE REVIEW

File: planner/core/logical_plan_builder.go [hunk 1]
```diff
@@ -3248,6 +3248,9 @@ func (b *PlanBuilder) buildUpdate(ctx context.Context, update *ast.UpdateStmt) (
 		if t.TableInfo.IsView() {
 			return nil, errors.Errorf("update view %s is not supported now.", t.Name.O)
 		}
+		if t.TableInfo.IsSequence() {
```
Comment (zimulala) [inline planner/core/logical_plan_builder.go:3251]: Could we check it in preprocess.go?

Comment (AilinKid) [reply planner/core/logical_plan_builder.go:3251]: Yes, we can.
I handle it like what the view does.

**Comment by Deardrops** [comment]
Do we need throw error when do some DDL on a sequence (table) ?

**Comment by AilinKid** [comment]
> Do we need throw error when do some DDL on a sequence (table) ?
yes we should, DDL case has been done in PR  https://github.com/pingcap/tidb/pull/16625 https://github.com/pingcap/tidb/pull/16605 and so on.

**Pull request merged**